support-frontend/test/services/pricing/PriceSummaryServiceSpec.scala

```diff
@@ -89,7 +89,7 @@ class PriceSummaryServiceSpec extends AsyncFlatSpec with Matchers {
       .find(_.promoCode == discountPromoCode)
       .value
       .discountedPrice shouldBe Some(166.50)
-    guardianWeekly(Europe)(RestOfWorld)(NoProductOptions)(Annual)(EUR).price shouldBe 270
+    guardianWeekly(Europe)(Domestic)(NoProductOptions)(Annual)(EUR).price shouldBe 318
```

Comment:
Isn't `GuardianWeekly` was only `Domestic` in the UK?

Reply:
No it's Domestic in all of the top level regions that we support (GB, US, CA, AU, NZ, EU), it's only ROW in the 'International' region